### ISSUE TITLE
Box Detector

### ISSUE THREAD

**Issue by acwatt** [acwatt/skybees#6]
issue for the object detection tensorflow model to find bee boxes in images

**Comment by acwatt** [comment]
# Notes from meeting with RC, DM 2021-07-22
Data storage:
- when I download a set of images, try to get minimum coverage to do the paper and save the images to an S3 bucket so even if we end up in negotiations with google 
	- look at storage costs and storage amount for a county size set of image	


EXTENSION: Look for an image resolution tool - good for setting the zoom level but also good as an important input to the covariates on the barcode

parameters to play with once we have more data:
- threshold


between apiary and box detectors: 
 Extension: snip away image outside the apiary bounding box and feed that to the box detector
 QA step: save in boxes dataframe: an indicator for if the boxes were found in apiary bounding boxes and if the apiary bounding box was found at all in the image.
Extension: barcode includes ag census or BIP number of hives/apiaries in this zip or county or state (high loss and high # of apiaries could tell us about the different patterns in the images -- look for apiaries that are more spares, etc)

NEXT:
- Get quickest rough county result (know lower bound, could get better but at least this good)
- Add barcode and have a flag 
 - add a flag for the width of pixels and location on the image (on, around, on the side)
- Select county from GitHub data
- finish GIS, download latlons for county
	- select desirable land types + a buffer of a few hundred feet
	- or just first cut into squares first, and keep any square/latlon that has 
- create more training data for classifier and apiary detector
- create a dataset, do sample stats and merge with GitHub data to address missingness by weighting, etc.


Wait on the paper draft until we have something to be inspired by (the shortcomings and the strengths of the results)

Once the county is complete: go get honey report, ag census, BIP survey for this county, and how does ours compare?
 - could be 80% of PA and other more standardized states, but much above other less-regulated states.
